diesel_derives/Cargo.toml

```diff
@@ -10,9 +10,9 @@ repository = "https://github.com/diesel-rs/diesel/tree/master/diesel_derives"
 autotests = false
 
 [dependencies]
-syn = { version = "0.15.0", features = ["full", "fold"] }
-quote = "0.6.0"
-proc-macro2 = "0.4.0"
+syn = { version = "1", features = ["full", "fold"] }
+quote = "1"
+proc-macro2 = "1"
```

Comment:
IMO: these requirements' range is a bit wide. Should be `1.0`?

Reply:
It would do the same thing. 1.0 will happily pull 1.1.0 since they're semver compatible.

Reply:
Ah, I forgot it! Looks good.